README.md

```diff
@@ -48,11 +48,20 @@ This method creates a new `%SegmentIterator%` over the input string, which will
 ### `%SegmentIterator%`
 
 This class iterates over segment boundaries of a particular string.
+
+#### Iteration result data
+
+* `index` is the code point index immediately following the last found boundary.
+* `segment` is the substring between the last found boundary and its preceding boundary.
+* `breakType` is a value characterizing `segment` and its terminating boundary. Some particularly important values are:
+  * For `word` granularity, "word" for letter/number/ideograph segments vs. "none" for spaces/punctuation/etc.
+  * For `line` granularity, "hard" for mandatory line breaks such as follow U+000A LINE FEED (`\n`) vs. "soft" for line break opportunities such as follow U+0020 SPACE.
```

Comment:
The `line` granularity is removed from the spec in Jan. So please remove this part.